BoxContentSDK/BoxContentSDK/Models/BOXRepresentation.m

```diff
@@ -41,27 +46,26 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
                                                                     nullAllowed:NO];
         self.details = detailsJSON;
         
-        NSDictionary *linksJSON = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyLinks
+        NSDictionary *contentJSON = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyContent
                                                                  inDictionary:JSONResponse
                                                               hasExpectedType:[NSDictionary class]
                                                                   nullAllowed:NO];
```

Comment:
the new xcode 8.3 isn't play nice. Seems to be centering our code. Will leave as is.